test/Test.hs

```diff
@@ -0,0 +1,33 @@
+module Main where
+
+import Data.Foldable (for_)
+import System.Directory (listDirectory, removeDirectoryRecursive, removeFile)
+import System.Exit (exitFailure)
+import System.FilePath (isExtensionOf, splitExtension, (<.>), (</>))
+
+main :: IO ()
+main = do
+    files <- map ("test/Test" </>) <$> listDirectory "test/Test/"
+    let testFiles = filter ("test" `isExtensionOf`) files
+    -- compare outputs of .test and .golden files
+    result <- traverse compareWithGolden testFiles >>= pure . foldr (&&) True
```

Comment:
Btw, I think you can use `relude` in `test-suite` only if add `{-# LANGUAGE NoImplicitPrelude #-}` on top of this module and then add `import Relude`. Because instead of this function you can then use `allM` from `Relude`:

* http://hackage.haskell.org/package/relude-0.1.0/docs/Relude-Foldable-Fold.html#v:allM

Since this is a test stanza then having extra dependency is not a problem.

Though, on the other hand, we probably want to know what exact test is failing... But this is another issue and another problem to solve.

Reply:
We can bring this all later, I guess. But, yeah, that's useful function!